Npm install fails with EPEERINVALID

`npm install` breaks error

```
npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/Users/andrey/.nvm/v0.10.32/bin/npm" "install"
npm ERR! node v0.10.32
npm ERR! npm  v2.1.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package less does not satisfy its siblings' peerDependencies requirements!

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/andrey/Projects/react/test/npm-debug.log
```

more detailed log

```
72 info install react-starter-kit@0.0.0
73 info postinstall react-starter-kit@0.0.0
74 info prepublish react-starter-kit@0.0.0
75 verbose stack Error: The package less does not satisfy its siblings' peerDependencies requirements!
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/lib/install.js:103:32
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/lib/install.js:225:5
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/read-installed/read-installed.js:138:5
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/read-installed/read-installed.js:251:14
75 verbose stack     at cb (/Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/read-installed/read-installed.js:251:14
75 verbose stack     at cb (/Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/read-installed/read-installed.js:251:14
75 verbose stack     at cb (/Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
75 verbose stack     at /Users/andrey/.nvm/v0.10.32/lib/node_modules/npm/node_modules/read-installed/read-installed.js:251:14
76 verbose cwd /Users/andrey/Projects/react/test
77 error Darwin 14.0.0
78 error argv "node" "/Users/andrey/.nvm/v0.10.32/bin/npm" "install"
79 error node v0.10.32
80 error npm  v2.1.12
81 error code EPEERINVALID
82 error peerinvalid The package less does not satisfy its siblings' peerDependencies requirements!
83 verbose exit [ 1, true ]
```

--- a/package.json
+++ b/package.json
@@ -43,7 +43,7 @@
     "jshint-stylish": "^1.0.0",
     "jsx-loader": "^0.12.2",
     "less": "^2.1.1",
-    "less-loader": "^0.7.8",
+    "less-loader": "^2.0.0",
     "merge-stream": "^0.1.6",
     "minimist": "^1.1.0",
     "protractor": "^1.5.0",

